### ISSUE TITLE
[A11y] Desktop Advanced Search: twitter icon

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1874]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Advanced Search

### Admin user?

No

### Issue Description

Advanced Search: https://staging.oregondigital.org/advanced

The twitter icon is missing the accessible name it is just reading the user handle.

### Suggested Fix

Add the accessible name, possible "twitter handle" if that is the correct term for the user handle.

### Can anybody QA or does this need an expert?

Anybody
